codecov.yml

```diff
@@ -0,0 +1,13 @@
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
```

Comment:
Can we also add the setting to ignore the generated `api/v1alpha1/zz_generated.deepcopy.go` file? Example from the release-service: https://github.com/redhat-appstudio/release-service/blob/main/codecov.yml#L2-L3.

Reply:
okay, will add.

Reply:
thanks!